depletion/casl_chain.py

```diff
@@ -0,0 +1,280 @@
+# This dictionary contains the 255-nuclides, simplified burnup chain used in
```

Comment:
This file currently has executable permissions, even though it doesn't need them
```
$ ls -l depletion/casl_chain.py
-rwxr-xr-x 1 drew users 11K Feb  9 14:35 depletion/casl_chain.py*
```

Reply:
Thanks for pointing that out -- fixed now :heavy_check_mark: